sentences/da/_common.yaml

```
@@ -46,10 +106,176 @@ lists:
+    values:
+      - in: "åben"
+        out: "on"
+      - in: "lukket"
+        out: "off"
+  bs_gas_states:
+    values:
+      - in: "(detekteret|påvist|tilstedeværende)"
+        out: "on"
+      - in: "(ikke detekteret|ikke påvist|fraværende)"
+        out: "off"
+  bs_heat_states:
+    values:
+      - in: "varmt"
+        out: "on"
+      - in: "normalt"
+        out: "off"
+  bs_light_states:
+    values:
+      - in: "detekteret"
+        out: "on"
+      - in: "ikke detekteret"
+        out: "off"
+  bs_lock_states:
+    values:
+      - in: "ulåst"
+        out: "on"
+      - in: "låst"
+        out: "off"
+  bs_moisture_states:
+    values:
+      - in: "vådt"
+        out: "on"
+      - in: "tørt"
+        out: "off"
+  bs_motion_states:
+    values:
+      - in: "(detekteret|påvist)"
+        out: "on"
+      - in: "(ikke detekteret|ikke påvist)"
+        out: "off"
+  bs_occupancy_states:
+    values:
+      - in: "(detekteret|påvist|tilstedeværende)"
+        out: "on"
+      - in: "(ikke detekteret|ikke påvist|fraværende)"
+        out: "off"
+  bs_opening_states:
+    values:
+      - in: "åben"
+        out: "on"
+      - in: "lukket"
+        out: "off"
+  bs_plug_states:
+    values:
+      - in: "tilkoblet"
+        out: "on"
+      - in: "frakoblet"
+        out: "off"
+  bs_power_states:
+    values:
+      - in: "tændt"
+        out: "on"
+      - in: "slukket"
+        out: "off"
+  bs_presence_states:
+    values:
+      - in: "(hjemme|tilstedeværende)"
+        out: "on"
+      - in: "(væk|fraværende)"
+        out: "off"
+  bs_problem_states:
+    values:
+      - in: "fundet"
+        out: "on"
+      - in: "intet fundet"
+        out: "off"
+  bs_running_states:
+    values:
+      - in: "(kører|aktiveret)"
+        out: "on"
+      - in: "(stoppet|deaktiveret)"
+        out: "off"
+  bs_safety_states:
+    values:
+      - in: "farlig"
+        out: "on"
+      - in: "sikker"
+        out: "off"
+  bs_smoke_states:
+    values:
+      - in: "(detekteret|påvist|tilstedeværende)"
+        out: "on"
+      - in: "(ikke detekteret|ikke påvist|fraværende)"
+        out: "off"
+  bs_sound_states:
+    values:
+      - in: "(detekteret|påvist|tilstedeværende)"
+        out: "on"
+      - in: "(ikke detekteret|ikke påvist|fraværende)"
+        out: "off"
+  bs_tamper_states:
+    values:
+      - in: "(detekteret|pillet ved)"
+        out: "on"
+      - in: "(ikke detekteret|urørt)"
+        out: "off"
+  bs_update_states:
+    values:
+      - in: "opdatering tilgængelig"
+        out: "on"
+      - in: "opdateret"
+        out: "off"
+  bs_vibration_states:
+    values:
+      - in: "(detekteret|vibrerer)"
+        out: "on"
+      - in: "(ikke detekteret|vibrerer ikke)"
+        out: "off"
+  bs_window_states:
+    values:
+      - in: "åben"
+        out: "on"
+      - in: "lukket"
+        out: "closed"
+
 expansion_rules:
   al: "(alle | alt)"
   blæser: "blæser[(e|en|ne)]"
   garagedør: "garagedør[(en|e|ene)]"
```

Both "garagedør" and "garageport" represent the same device class, so I don't think they need to be treated as separate expansion rules.

```suggestion
  garagedør: "garage(dør|port)[(en|e|ene)]"
```